include/dxc/DxilContainer/RDAT_LibraryTypes.inl

```diff
@@ -438,67 +273,94 @@ RDAT_STRUCT_TABLE(NodeID, NodeIDTable)
+    RDAT_UNION_ELIF(MaxRecursionDepth,
+                    getAttribKind() ==
+                        hlsl::RDAT::NodeFuncAttribKind::MaxRecursionDepth)
       RDAT_VALUE(uint32_t, MaxRecursionDepth)
-    RDAT_UNION_ELIF(LocalRootArgumentsTableIndex, getAttribKind() == hlsl::RDAT::NodeFuncAttribKind::LocalRootArgumentsTableIndex)
+    RDAT_UNION_ELIF(
+        LocalRootArgumentsTableIndex,
+        getAttribKind() ==
+            hlsl::RDAT::NodeFuncAttribKind::LocalRootArgumentsTableIndex)
       RDAT_VALUE(uint32_t, LocalRootArgumentsTableIndex)
-    RDAT_UNION_ELIF(MaxDispatchGrid, getAttribKind() == hlsl::RDAT::NodeFuncAttribKind::MaxDispatchGrid)
+    RDAT_UNION_ELIF(MaxDispatchGrid,
+                    getAttribKind() ==
+                        hlsl::RDAT::NodeFuncAttribKind::MaxDispatchGrid)
       RDAT_INDEX_ARRAY_REF(MaxDispatchGrid)
     RDAT_UNION_ENDIF()
   RDAT_UNION_END()
 RDAT_STRUCT_END()
 #undef RECORD_TYPE
 
-
 #define RECORD_TYPE NodeShaderIOAttrib
 RDAT_STRUCT_TABLE(NodeShaderIOAttrib, NodeShaderIOAttribTable)
   RDAT_ENUM(uint32_t, hlsl::RDAT::NodeAttribKind, AttribKind)
   RDAT_UNION()
     RDAT_UNION_IF(ID, getAttribKind() == hlsl::RDAT::NodeAttribKind::OutputID)
       RDAT_RECORD_REF(NodeID, OutputID)
-    RDAT_UNION_ELIF(MaxRecords, getAttribKind() == hlsl::RDAT::NodeAttribKind::MaxRecords)
+    RDAT_UNION_ELIF(MaxRecords,
+                    getAttribKind() == hlsl::RDAT::NodeAttribKind::MaxRecords)
       RDAT_VALUE(uint32_t, MaxRecords)
-    RDAT_UNION_ELIF(MaxRecordsSharedWith, getAttribKind() == hlsl::RDAT::NodeAttribKind::MaxRecordsSharedWith)
+    RDAT_UNION_ELIF(MaxRecordsSharedWith,
+                    getAttribKind() ==
+                        hlsl::RDAT::NodeAttribKind::MaxRecordsSharedWith)
       RDAT_VALUE(uint32_t, MaxRecordsSharedWith)
-    RDAT_UNION_ELIF(RecordSizeInBytes, getAttribKind() == hlsl::RDAT::NodeAttribKind::RecordSizeInBytes)
+    RDAT_UNION_ELIF(RecordSizeInBytes,
+                    getAttribKind() ==
+                        hlsl::RDAT::NodeAttribKind::RecordSizeInBytes)
       RDAT_VALUE(uint32_t, RecordSizeInBytes)
-    RDAT_UNION_ELIF(RecordDispatchGrid, getAttribKind() == hlsl::RDAT::NodeAttribKind::RecordDispatchGrid)
+    RDAT_UNION_ELIF(RecordDispatchGrid,
+                    getAttribKind() ==
+                        hlsl::RDAT::NodeAttribKind::RecordDispatchGrid)
       RDAT_RECORD_VALUE(RecordDispatchGrid, RecordDispatchGrid)
-    RDAT_UNION_ELIF(OutputArraySize, getAttribKind() == hlsl::RDAT::NodeAttribKind::OutputArraySize)
+    RDAT_UNION_ELIF(OutputArraySize,
+                    getAttribKind() ==
+                        hlsl::RDAT::NodeAttribKind::OutputArraySize)
       RDAT_VALUE(uint32_t, OutputArraySize)
-    RDAT_UNION_ELIF(AllowSparseNodes, getAttribKind() == hlsl::RDAT::NodeAttribKind::AllowSparseNodes)
+    RDAT_UNION_ELIF(AllowSparseNodes,
+                    getAttribKind() ==
+                        hlsl::RDAT::NodeAttribKind::AllowSparseNodes)
```

Comment:
This run of multiline macros makes me wonder if formatting could be reconsidered here too.